app/views/foreman_ansible/job_templates/run_playbook-ansible_default.erb

```diff
@@ -12,6 +12,14 @@ template_inputs:
   description: The playbook to run against given hosts
```

Comment:
I don't think this is the right job to implement this for. This one is to run arbitrary playbook. While it may be handy there too, I think we need it mainly for https://github.com/theforeman/foreman_ansible/blob/master/app/views/foreman_ansible/job_templates/ansible_roles_-_ansible_default.erb

That job template is used to run all assigned roles of the host/host group.